### PR TITLE
Misc cleanups

### PR DESCRIPTION
**Pull request by EdHastingsCasperAssociation**
This PR cleans up a few things, re-enables some tests, fixes a minor bug. A few tests need to be looked at by SME's.

### CODE REVIEW

File: execution_engine_testing/tests/src/test/contract_messages.rs [hunk 1]
```diff
@@ -101,12 +99,12 @@ fn upgrade_messages_emitter_contract(
     )
     .build();
 
-    let new_topics = BTreeMap::from([(
-        MESSAGE_EMITTER_GENERIC_TOPIC.to_string(),
-        MessageTopicOperation::Add,
-    )]);
+    // let new_topics = BTreeMap::from([(
```
Comment (zajko) [inline execution_engine_testing/tests/src/test/contract_messages.rs:102]: Should we remove commented code?

File: node/src/components/block_validator/state.rs [hunk 2]
```diff
@@ -370,11 +370,13 @@ impl BlockValidationState {
                             "no further missing transactions - block validation complete"
                         );
                         let new_state = BlockValidationState::Valid(appendable_block.timestamp());
+                        println!("OK {:?}", responders);
```
Comment (zajko) [inline node/src/components/block_validator/state.rs]: Are these prints intentional?

**Comment by darthsiroftardis** [comment]
@EdHastingsCasperAssociation  I've looked at the failing tests, and have cut a PR to your branch

**Review by zajko — APPROVED** [review]

File: node/src/components/block_validator/state.rs (hunk 2 above)
Comment (EdHastingsCasperAssociation) [reply node/src/components/block_validator/state.rs]: i left it there to call attn for Karan...the numbers are questionable

File: execution_engine_testing/tests/src/test/contract_messages.rs (hunk 1 above)
Comment (EdHastingsCasperAssociation) [reply execution_engine_testing/tests/src/test/contract_messages.rs:102]: i'm less picky about it in test code...particularly when cleaning up other peoples code as this PR is doing...the original dev in their glorious SME-ness thought it useful to dump that data to stdout while working on the initial feature, and we may find it useful as well when we continue the work on it in their absence. its harmless, and we can always remove it later if it proves to be irrelevant.

**Pull request merged**